syntax="proto3";

package simple_databroker;

message BrokerRequest{
    int32 id = 1;
    float sensor1 = 2 ;
    float sensor2 = 3 ;
    float sensor3 = 4 ;
    float sensor4 = 5 ;
}

message BrokerResponse{
    int32 id = 1;
    bool prediction = 2;
}

service BrokerService{
    rpc SimpleMethod( BrokerRequest) returns ( BrokerResponse);
}